gnucash/import-export/ofx/gnc-ofx-import.c

```diff
@@ -1038,6 +1039,54 @@ double ofx_get_investment_amount(const struct OfxTransactionData* data)
     }
 }
 
+// This callback is called when the user is done matching transactions.
+static void
+gnc_ofx_match_done (GtkDialog *dialog, gint response_id, gpointer user_data)
+{
+    ofx_info* info = (ofx_info*) user_data;
+    GSList *iter = NULL;
+    
+    if (info->statement == NULL)
+        return;
+    
+    if (response_id == GTK_RESPONSE_OK)
+    {
+        gint response = GTK_RESPONSE_NONE;
+        // Prepare a dialog to ask the user whether they want to open a reconcile window.
+        GtkWidget* dialog = gtk_message_dialog_new ( info->parent,
+                                                    GTK_DIALOG_MODAL | GTK_DIALOG_DESTROY_WITH_PARENT,
+                                                    GTK_MESSAGE_WARNING, GTK_BUTTONS_NONE,
+                                                    "%s", _("Your statement(s) include(s) balance information,\ndo you want to reconcile now?"));
```

Comment:
Personally I do not like this, can it be changed to ngettext(3) message so you have "Your statement includes..." and "Your statements include..."